### PR TITLE
drpc: fix a typo

### PR DESCRIPTION
**Pull request by raghavendra-talur**

### CODE REVIEW

File: controllers/drplacementcontrol.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ const (
 )
 
 var (
-	WaitForAppResouceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
+	WaitForAppResourceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
```
Comment (nirs) [inline controllers/drplacementcontrol.go:36]: If we touch this, maybe match the error message?

    WaitingForAppResourcesRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")

But it is very hard to work with such long names using camelCase style. We really much shorter names, how about:

    WaitingForRestore error = errorswrapper.New("Waiting for App resources to be restored...")

**Review by ELENAGER — APPROVED** [review]
I am fine with this change

**Review by nirs — APPROVED** [review]
Lets fix the typo first and make other improvements later.

**Review by BenamarMk — APPROVED** [review]

**Comment by BenamarMk** [comment]
@raghavendra-talur I can merge if you fix the linter error

File: controllers/drplacementcontrol.go [hunk 2]
```diff
@@ -33,11 +33,11 @@ const (
...
-	WaitForAppResouceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
-	WaitForVolSyncDestRepToComplete    error = errorswrapper.New("Waiting for VolSync RD to complete...")
-	WaitForSourceCluster               error = errorswrapper.New("Waiting for primary to provide Protected PVCs...")
-	WaitForVolSyncManifestWorkCreation error = errorswrapper.New("Waiting for VolSync ManifestWork to be created...")
-	WaitForVolSyncRDInfoAvailibility   error = errorswrapper.New("Waiting for VolSync RDInfo...")
+	WaitForAppResourceRestoreToComplete error = errorswrapper.New("Waiting for App resources to be restored...")
+	WaitForVolSyncDestRepToComplete     error = errorswrapper.New("Waiting for VolSync RD to complete...")
+	WaitForSourceCluster                error = errorswrapper.New("Waiting for primary to provide Protected PVCs...")
+	WaitForVolSyncManifestWorkCreation  error = errorswrapper.New("Waiting for VolSync ManifestWork to be created...")
+	WaitForVolSyncRDInfoAvailibility    error = errorswrapper.New("Waiting for VolSync RDInfo...")
```
Comment (nirs) [inline controllers/drplacementcontrol.go:40]: It is very bad that we cannot make one line fix without changing lines around it.

Can we avoid it by using larger indent so future changes in the names will not cause
unwanted change in other lines?

Is this enforced by by gofmt or by 3rd party linter?

**Pull request merged**